kofta/public/locales/de/translation.json

```diff
@@ -172,13 +172,13 @@
 			"togglePushToTalkKeybind": "toggle push-to-talk keybind"
 		},
 		"wsKilled": {
-			"description": "Websocket was killed by the server. This usually happens when you open the website in another tab.",
-			"reconnect": "reconnect"
+			"description": "Websocket wurden vom Server gekillt. Dies kann passieren, wenn du die Webseite in einem neuen Tab erneut öffnest.",
+			"reconnect": "Neu verbinden."
 		},
 		"deviceNotSupported": {
-			"notSupported": "Your device is currently not supported. You can create an",
-			"linkText": "issue on GitHub",
-			"addSupport": "and I will try adding support for your device."
+			"notSupported": "Dein Gerät ist zurzeit leider nicht unterstützt. Du kannst ein",
```

Comment:
"Dein Gerät **wird** zurzeit leider [...]"

Reply:
Stimmt, ist mir so nicht in den Sinn gekommen.